recipes/itsxpress/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "itsxpress" %}
+{% set version = "1.5.6" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "53f833545b230f116ac16417ee46dda5f1174f170c768192a9cd5056e1eba25e" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://files.pythonhosted.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - itsxpress=itsxpress.main:main
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
please use `pip install`

Reply:
I now use pip. Looking at other recipes this seemed to be the most common way:
python -m pip install --no-deps --ignore-installed .
If that's not correct please let me know.